### DISCUSSION TITLE
Practica CSS

### DISCUSSION THREAD

**Discussion by R-V-L** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#385] [Show and tell]
Seguramente tiene un monton de typos y codigo basura por hacer cambios a ultimo momento... pero es lo mas que mejor pude hacer. [https://vacunacion-launchx.herokuapp.com/](https://vacunacion-launchx.herokuapp.com/)

Nota: no apto para celulares

**Comment by OscarPedroza** [comment]
Muy buen trabajo compañero!, bastante similar al diseño.

**Comment by JuanSalvador73** [comment]
Va muy bien. En cuanto puedas ve la forma de hacerla responsive. Pero para la entrega reune los requisitos.
